citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -163,16 +168,34 @@ <h2>
                 $("#syncProgress").addClass("progress-bar-striped active");
                 $("#syncText").text(Math.round(percent) + "% synced");
         	}
+        	getCurrentItemsData();
         	setTimeout(pollStatus,1000);
         } else {
         	$("#syncProgress").attr('style', "width: 100%");
         	$("#syncProgress").attr('aria-valuenow', 100);
             $("#syncProgress").attr('aria-valuemax', 100);
             $("#syncProgress").removeClass("progress-bar-striped active");
             $("#syncText").text("Up to date");
+            getCurrentItemsData();
```

Comment:
since you call this in either block (if and else), you might as well call it afterwards just once